test/Bootstrap/ConsignmentTestCase.php

```diff
@@ -207,6 +207,14 @@ protected function doConsignmentTest(array $testData): void
         if ($testData[self::PACKAGE_TYPE] === AbstractConsignment::PACKAGE_TYPE_PACKAGE) {
             self::assertNotEmpty($consignment->getBarcode(), 'Barcode is missing');
         }
+
+        $carrier = $consignment->getCarrier();
+
+        if (CarrierInstabox::NAME === $carrier->getName()) {
```

Comment:
Dit hoeft helemaal niet, je moet er gewoon voor zorgen dat de functie wordt aangeroepen in/door een test. nu heb je super specifieke business logic in de default test die niet nodig is. 

Dat die line niet al gecovered werd laat je eigenlijk zien dat je wat mist in je codes. Er zou een Validator Rule moeten komen op consignment en/of order die checkt of er een dropoff point is als het verplicht is.